tests/tests_fabric/accelerators/test_tpu.py

```diff
@@ -25,9 +25,10 @@ def test_auto_device_count():
     assert TPUAccelerator.auto_device_count() > 1
 
 
-@RunIf(tpu=True)
-def test_availability():
-    assert TPUAccelerator.is_available()
```

Comment:
I removed this test because `@RunIf(tpu=True)` will only let this test run if `TPUAccelerator.is_available()` which is exactly what we test in the test definition. Meaning it's as silly as:

```
@if(1 == 1):
def test_availability():
    assert 1 == 1
```